L5_Application/mp3_struct.h

```diff
@@ -0,0 +1,20 @@
+#ifndef MP3INFO_H_
+#define MP3INFO_H_
+
+#include <stdio.h>
+#include <string.h>
+#include "ff.h"
+
+class MetaData {
+private:
+  char *songs[100];
```

Comment:
Might to too many arrays too keep in memory, minimum 4K needed for this implementation.